modules/oauth/src/main/java/org/mule/extension/oauth2/internal/AbstractGrantType.java

```diff
@@ -21,4 +37,8 @@ public static String buildAuthorizationHeaderContent(String accessToken) {
     return "Bearer " + accessToken;
   }
 
+  @Override
+  public void setMuleContext(MuleContext muleContext) {
+    this.muleContext = muleContext;
```

Comment:
For some reason, the muleContext is not being automatically injected into this object. The context is being set in https://github.com/mulesoft/mule/pull/3836/files#diff-bf73dd2c2b906c68a00d1d8ec0c1d7d3R138